websurfx/config.lua

```diff
@@ -47,6 +47,7 @@ theme = "simple" -- the theme name which should be used for the website
 
 -- ### Caching ###
 redis_url = "redis://127.0.0.1:8082" -- redis connection url address on which the client should connect on.
+in_memory_cache = true
```

Comment:
```suggestion
```
I would suggest not having this option at all, I feel that having a feature based approach would be better :slightly_smiling_face: .

Reply:
That wont be too consistent.
In this setup, one can build the app, which supports:
* caching in Redis - when a `redis_url` is specified
* caching in process - when a `redis_url` is not specified, but the `in_memory_cache` set to true
* no caching - when there is no `redis_url` and `in_memory_cache` set to false